egs/librispeech/ASR/pruned_transducer_stateless7/zipformer.py

```diff
@@ -1458,7 +1468,11 @@ def forward(self, x: Tensor, key_padding_mask: Optional[Tensor] = None):
            a Tensor of shape (1, N, C)
         """
         if key_padding_mask is not None:
-            pooling_mask = key_padding_mask.logical_not().to(x.dtype)  # (N, T)
+            if torch.jit.is_tracing():
```

Comment:
Is `logical_not` not supported by `onnx`?

Could you try
```python3
pooling_mask = (~key_padding_mask).to(x.dtype)
```

Reply:
Yes, not supported